examples/remote_execution/buildbarn/README.md

```diff
@@ -1,30 +1,51 @@
-## Remote execution integration with Build Barn
+## Remote execution integration with Buildbarn
 
-This project provides a small example of what a project that utilizes [Build Barn](https://github.com/buildbarn) for RE might look like.
+This project provides a small example of what a project that utilizes [Buildbarn](https://github.com/buildbarn).
 
 In this document, we will go over the key configs used in this setup.
+Using a local docker-compose deployment from the [example deployment repo](https://github.com/buildbarn/bb-deployments)
 
-### Relevant configs in .buckconfig
+### Deploy Buildbarn
 
-First, the Build Barn endpoint and certificate should be configured as the following:
+This is mostly a turn-key solution, with just one modification.
```

Comment:
Let's maybe just remove this and wait for #181 to get merged.

Reply:
Yes! Great point. I originally wrote this last week, when we had no timeline for a solution.
And then an in-review Pull Request would be a good location for other confused developers to find the information. I planned to remove this after the solution, so the final artifact would not have it.

I'm fine with wating for 181, so I can remove the mention from here.